actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -437,6 +442,9 @@ def link_to_if(condition, name, options = {}, html_options = {}, &block)
       # Creates a mailto link tag to the specified +email_address+, which is
       # also used as the name of the link unless +name+ is specified. Additional
       # HTML attributes for the link can be passed in +html_options+.
+      # Note the order of params, which is reversed compared to +link_to+. Provide
+      # the part that is used for the href first, and the part that is visible
+      # to users second.
```

Comment:
@senny Yes, I find this worth pointing out. For me consistency between the link-helper is very important. So places that break this consistency should be pointed out. This is only stressed by the fact that the helper works the same way as link_to when it comes to block elements. I don't agree that the examples are a replacement for the text documentation. Following this logic there is no need to write text at all.